css/style.css

```diff
@@ -0,0 +1,180 @@
+*{
+    margin:0;
+    padding: 0;
+    box-sizing: border-box;
+}
+li {
+    list-style: none;
+
+}
+  
+a {
+    color: #000;
+    text-decoration: none;
+}
+
+body {
+    background: whitesmoke;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+body > h1 {
+    text-align: center;
+    font-size: large;
+    font-style: oblique;
+}
+#header {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    
+}
+  #header-img {
+    margin-top: 5px;
+    width: 200px;
+    height: 200px;
+    border-radius: 50%;
+}
+  
+  #logo {
+    display: flex;
+    align-items: center;
+}
+  #logo > span {
+    margin-left: 20px;
+    font-size: 30px;
+    font-style: normal;
+    text-transform:uppercase;
+    text-decoration: overline;
+}
+  #nav-bar> ul{
+    margin-top: 15px;
+    list-style-type: none;
+}
+  #nav-bar> ul> li {
+    float: left;
+    border-right: 1px solid black;
+    padding-right: 10px;
+    margin-bottom: 20px;
+    margin-right: 100px;
+    
+}
+  
+  nav > ul >li >a {
+    display: block;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: black;
+    font-size: 20px;
+}
+  nav > ul >li > a:hover{
+    text-decoration: underline;
+}
+  nav > ul >li > a:active{
+    text-decoration: underline;
+}
+#form{
+  display:flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 100px;
+  
+}
+.form {
+  margin-top:20px;
+}
+#submit {
+  border-radius: 35%;  
+  text-transform: uppercase;
+  background: #2381ad;
+  padding: 10px;
+}
+main{
+  margin-top: 20px;
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+#feature {
+  max-width: 90%;
+  height: 200px;
+  display: flex;
+  flex-direction:column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+.grid{
+  display: flex;
+}
+.grid1{
+  margin-right: 87px;
+}
+.grid2 {
+  margin-left: 100px;
+}
+#quality{
+    margin-left: 19px;
```

Comment:
Please make sure you have a container for the features section  and use flexbox or grid to align the different features.
Using diffrent `margin` for the diffrent features makes your code not so practical.
Kindly make the changes, @michael-duke